tmux.1

```diff
@@ -5706,9 +5706,17 @@ otherwise by
 .Ql ||
 and
 .Ql &&
-evaluate to true if either or both of two comma-separated alternatives are
+evaluate to true if any or all of the comma-separated alternatives are
 true, for example
 .Ql #{||:#{pane_in_mode},#{alternate_on}} .
+.Ql !
```

Comment:
This is rendering as `‘’!` for me. Do you know off the top of your head how to fix that? If not, I'll try to figure it out.

Reply:
I'm not sure what you mean... it shows 0 or 1 for me with and without your changes?

Reply:
Oh you meand the mdoc, duh.

Reply:
It works for me with both mandoc and nroff:
~~~~
$ nroff -mdoc tmux.1|grep '||:#{pane_in_mode'
       ‘#{||:#{pane_in_mode},#{alternate_on}}’.
$ mandoc tmux.1|grep '||:#{pane_in_mode'
     ‘#{||:#{pane_in_mode},#{alternate_on}}’.
~~~~

Reply:
Oh duh again, you mean the `.Ql !`. You need to escape it `.Ql \&!`.

Reply:
But why do you want a `!` there anyway?

Reply:
OK it would help if I read the man page changes. I'll just add the `\&`.